build.sbt

```diff
@@ -14,19 +14,18 @@ javacOptions in doc := Seq("-source", "1.7")
 
 resolvers += "Twitter" at "http://maven.twttr.com/"
 
-resolvers += "Finch.io" at "http://repo.konfettin.ru"
-
 libraryDependencies ++= Seq(
-  "io"              %% "finch"              % "0.1.6",
-  "com.twitter"     %% "finagle-stats"      % "6.22.0",
-  "com.twitter"     %% "finagle-core"       % "6.22.0",
-  "com.twitter"     %% "twitter-server"     % "1.8.0",
-  "org.slf4j"       %  "slf4j-api"          % "1.7.7",
-  "org.slf4j"       %  "jul-to-slf4j"       % "1.7.7",
-  "org.slf4j"       %  "jcl-over-slf4j"     % "1.7.7",
-  "org.slf4j"       %  "log4j-over-slf4j"   % "1.7.7",
-  "ch.qos.logback"  %  "logback-classic"    % "1.1.2"   % "test",
-  "org.scalatest"   %% "scalatest"          % "2.2.2"   % "test"
+  "com.github.finagle"  %% "finch-core"         % "0.3.0",
```

Comment:
Yeah, I've have this branch for a few weeks.  I was waiting to have time to try and figure out how to do response compression with the new httpx server builder. I found out that currently there isn't a param for it but I might be able to contribute it based on this conversation on finaglers https://groups.google.com/forum/#!topic/finaglers/nE-7slWsfUM

Since it's not there I'll probably end up updating to 0.4.0 with a note about the regression and then re-enable once its in finagle.